src/main/java/uk/gov/hmcts/ccd/data/user/DefaultUserRepository.java

```diff
@@ -76,10 +75,11 @@ public IDAMProperties getUserDetails() {
     @Override
     public Set<SecurityClassification> getUserClassifications(String jurisdictionId) {
         Set<String> roles = this.getUserRoles();
+        final List<String> filteredRoles = roles.stream()
```

Comment:
I've seen this filtering of roles on many other places.
Can we introduce a method getJurisdictionRoles(String jurisdiction) and move this code in there?

Reply:
Also, we can further improve performance by excluding roles of type -loa* . Can we introduce a method 
getJurisdictionRolesExcludingLoa and use that?

Reply:
Actually there's already a filterRoles private method used a little below that line, but I think it's filtering roles with `caseworker-JURISTDICTION` pattern and not filtering out the `-loa` roles...